### ISSUE TITLE
NoPlayerPositionSetException() after re-logging in

### ISSUE THREAD

**Issue by Calcyfer** [PokemonGoF/PokemonGo-Bot#4358]
### Expected Behavior
Metrics should be shown after re-login

### Actual Behavior
NoPlayerPositionSetException() after re-logging in

### Output when issue occurred
```
2016-08-20 15:18:23,154 [PokemonGoBot] [INFO] [api_error] Session stale, re-logging in.
2016-08-20 15:18:23,171 [PokemonGoBot] [INFO] [login_started] Login procedure started.
Traceback (most recent call last):
  File "pokecli.py", line 674, in <module>
    main()
  File "pokecli.py", line 176, in main
    report_summary(bot)
  File "pokecli.py", line 211, in report_summary
    metrics.capture_stats()
  File "pokemongo_bot\metrics.py", line 84, in capture_stats
    response_dict = request.call()
  File "pokemongo_bot\api_wrapper.py", line 217, in call
    if not self.can_call():
  File "pokemongo_bot\api_wrapper.py", line 118, in can_call
    raise NoPlayerPositionSetException()
pgoapi.exceptions.NoPlayerPositionSetException
```

### Steps to Reproduce
Get a stale session

### Other Information
OS: 
Windows 10
Branch: 
dev
Git Commit: 
cc3b4c92b94b9e6b8f23f08eae93189a82d5cfd0
Python Version: 
2.7

**Comment by net8q** [comment]
Confirmed here, there's a double exception during the summary report, original exception is : 

>   File "./pokecli.py", line 677, in <module>
    main()
  File "./pokecli.py", line 118, in main
    bot.tick()
  File "./pokemongo_bot/__init__.py", line 538, in tick
    self.check_session(self.position[0:2])
  File "./pokemongo_bot/__init__.py", line 690, in check_session
    self.login()
  File "./pokemongo_bot/__init__.py", line 714, in login
    str(self.config.password)):
  File "./pokemongo_bot/api_wrapper.py", line 77, in login
    ret_value = PGoApi.login(self, *args)
  File "pgobot/src/pgoapi/pgoapi/pgoapi.py", line 179, in login
    response = self.app_simulation_login()
  File "pgobot/src/pgoapi/pgoapi/pgoapi.py", line 156, in app_simulation_login
    response = request.call()
  File "pgobot/src/pgoapi/pgoapi/pgoapi.py", line 215, in call
    raise NoPlayerPositionSetException()
pgoapi.exceptions.NoPlayerPositionSetException

**Comment by Calcyfer** [comment]
Looking at the code, it seems the altitude is missing after the relog, I'll check out if I can fix that.

**Comment by mjmadsen** [comment]
Should be fixed with #4364 on dev branch.
